README.md

```diff
@@ -75,6 +75,7 @@ Create ```.env``` file in a root directory or set environment variables with fol
 
 - `API_URI` (required) - URI of a running instance of Saleor GraphQL API.
   If you are running Saleor locally with the default settings, set `API_URI` to: `http://localhost:8000/graphql/`.
+  If you are running Saleor from cloud make sure that you add `/` at the end of `API_URI`.
```

Comment:
It's not specific to Cloud, there is always a slash at the end.

Reply:
Fixed to make it more generic 😀